easylistportuguese/easylistportuguese_specific_block.txt

```diff
@@ -97,4 +97,5 @@
 ||ecoregional.com.br/wp-content/uploads/*/jardim-pq-*.jpg
 ||ecoregional.com.br/wp-content/plugins/icegram/
 ||notebookcheck.info/fileadmin/Sonstiges/ama_abn2_$xmlhttprequest
-||noticiasdecoimbra.pt/*/modal/*/modal.min*^$stylesheet,script
\ No newline at end of file
+||noticiasdecoimbra.pt/*/modal/*/modal.min*^$stylesheet,script
+||i1.wp.com/www.cenariomt.com.br/wp-content/uploads/*/provincia-at.png
```

Comment:
@felippesantos46  could you remove this filter from this PR, it belongs to a different one.

https://github.com/easylist/easylistportuguese/pull/76